laravel-t-shirts/app/Console/Commands/CheckResponseStatusesCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Psr\SimpleCache\CacheInterface;
+
+class CheckResponseStatusesCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:responses:status';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Check responses statuses from the cache.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle(CacheInterface $cache)
+    {
+        $cache->set('statuses', [200 => 5, 204 => 2, 301 => 1, 302 => 2, 400 => 50, 500 => 2]);
```

Comment:
It was as example, in general in command you should not have setting of cache value.